core/network/types/status.hpp

```diff
@@ -6,15 +6,15 @@
 #ifndef KAGOME_CORE_NETWORK_TYPES_STATUS_HPP
 #define KAGOME_CORE_NETWORK_TYPES_STATUS_HPP
 
-#include <vector>
 #include <algorithm>
 #include <libp2p/peer/peer_info.hpp>
+#include <vector>
 
 #include "network/types/roles.hpp"
 #include "primitives/common.hpp"
 
-using kagome::primitives::BlockNumber;
 using kagome::primitives::BlockHash;
```

Comment:
Please don't use using declarations in a global namespace

Reply:
Ok